Libplanet.Explorer/Controllers/ExplorerController.cs

```diff
@@ -36,6 +39,129 @@ public BlockChain<T> GetBlockChain()
+            {
+                Field<StringGraphType>(
+                    "Signer",
+                    resolve: ctx => ctx.Source.Signer.ToString()
+                );
+                Field<StringGraphType>(
+                    "PublicKey",
+                    resolve: ctx => ByteUtil.Hex(ctx.Source.PublicKey.Format(true))
+                );
+                Field<ListGraphType<StringGraphType>>(
+                    "UpdatedAddress",
+                    resolve: ctx => from address in ctx.Source.UpdatedAddresses select address.ToString()
+                );
+                Field<StringGraphType>(
+                    "Signature",
+                    resolve: ctx => ByteUtil.Hex(ctx.Source.Signature)
+                );
+                Field(x => x.Timestamp);
+                Field<ListGraphType<IActionType>>("Actions");
+            }
+        }
+
+        public class IActionType : ObjectGraphType<IAction>
+        {
+            public IActionType()
+            {
+                Field<ListGraphType<PlainValueKeyValuePairType> >(
+                    "PlainValue",
+                    resolve: ctx =>
+                    {
+                        List<PlainValueKeyValuePair> result = new List<PlainValueKeyValuePair>();
+                        foreach(KeyValuePair<string, object> item in ctx.Source.PlainValue)
+                        {
+                            result.Add(new PlainValueKeyValuePair(){
+                                Key = item.Key,
+                                Value = item.Value
+                            });
+                        }
+                        return result;
+                    }
+                );
+            }
+        }
+
+        public class PlainValueKeyValuePairType : ObjectGraphType<PlainValueKeyValuePair>
+        {
+            public PlainValueKeyValuePairType()
+            {
+                Field<StringGraphType>("Key");
+                Field<StringGraphType>("Value");
+            }
+        }
+
+        public class BlocksQuery : ObjectGraphType
+        {
+            private BlockChain<T> _chain;         
+            public BlocksQuery(BlockChain<T> chain)
+            {
+                Field<ListGraphType<BlockType>>(
+                    "blocks",
+                    resolve: context => chain
+                );
+
+                Field<BlockType>(
+                    "block",
+                    arguments: new QueryArguments(
+                        new QueryArgument<IdGraphType> { Name = "index" }
+                    ),
+                    resolve: context =>
+                    {
+                        ulong index = context.GetArgument<ulong>("index");
+                        return _chain.FirstOrDefault(x => (ulong)x.Index == index);
+                    }
+                );
+                _chain = chain;
+            }
+        }
+        
+        [HttpGet("/graphql/")]
+        public IActionResult getGraphQLResult(
```

Comment:
Although existing code stick with this Java-style convention, could you rename it to `GetGraphQLResult`?  (We should rename other existing names too.)